{"uuid": "5b11db9b-3fd4-4e87-870c-3cb6e4d079ef", "children": ["273eb421-9ade-412e-8b2d-402d994ff072", "8f58e62b-630c-42b4-b8f1-31dc915ccbae", "1cd8e721-5da8-4d79-b388-0ba7886c4522", "321e7bb7-a904-4413-a080-cb06c255798d", "90d7bae1-d89a-4542-bab1-edf5f3391f4f", "e61371d2-ea21-458d-b0b3-32ec9c2c09f8", "601ef2fe-c619-4c13-a8dd-c16c72694b64", "ffc9d4d4-f3b2-4f32-8021-15911ba1c847", "99a8c418-9948-40e0-a69e-bd7e309e0f87", "af645397-7a42-4817-8cc5-8e76224c2f4a"], "befores": [{"name": "ops_test", "status": "broken", "statusDetails": {"message": "juju.errors.JujuConnectionError: Model not found: admin/test\n", "trace": "  File \"/home/yanks/Documents/canonical/jenkins-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 80, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/yanks/Documents/canonical/jenkins-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/home/yanks/Documents/canonical/jenkins-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 902, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n  File \"/home/yanks/Documents/canonical/jenkins-k8s-operator/.tox/integration/lib/python3.10/site-packages/pytest_asyncio/plugin.py\", line 304, in _asyncgen_fixture_wrapper\n    result = event_loop.run_until_complete(setup())\n  File \"/usr/lib/python3.10/asyncio/base_events.py\", line 649, in run_until_complete\n    return future.result()\n  File \"/home/yanks/Documents/canonical/jenkins-k8s-operator/.tox/integration/lib/python3.10/site-packages/pytest_asyncio/plugin.py\", line 286, in setup\n    res = await gen_obj.__anext__()\n  File \"/home/yanks/Documents/canonical/jenkins-k8s-operator/.tox/integration/lib/python3.10/site-packages/pytest_operator/plugin.py\", line 231, in ops_test\n    await ops_test._setup_model()\n  File \"/home/yanks/Documents/canonical/jenkins-k8s-operator/.tox/integration/lib/python3.10/site-packages/pytest_operator/plugin.py\", line 719, in _setup_model\n    model_state = await self._connect_to_model(\n  File \"/home/yanks/Documents/canonical/jenkins-k8s-operator/.tox/integration/lib/python3.10/site-packages/pytest_operator/plugin.py\", line 678, in _connect_to_model\n    await model.connect(state.full_name)\n  File \"/home/yanks/Documents/canonical/jenkins-k8s-operator/.tox/integration/lib/python3.10/site-packages/juju/model.py\", line 675, in connect\n    _, model_uuid = await self._connector.connect_model(model_name, **kwargs)\n  File \"/home/yanks/Documents/canonical/jenkins-k8s-operator/.tox/integration/lib/python3.10/site-packages/juju/client/connector.py\", line 164, in connect_model\n    raise JujuConnectionError('Model not found: {}'.format(_model_name))\n"}, "start": 1711364782320, "stop": 1711364782436}], "start": 1711364782320, "stop": 1711364782447}